avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDe.java

```diff
@@ -41,6 +41,7 @@
 
   private static final Map<String, Schema> primitiveSchemas;
   protected SchemaRegistryClient schemaRegistry;
+  protected boolean autoRegister;
```

Comment:
Why is this field in `AbstractKafkaAvroSerDe` and not `AbstractKafkaAvroSerializer`?